transport/uml/push/register_deviceToken.puml

```diff
@@ -0,0 +1,31 @@
+@startuml
+!theme bluegray
+!pragma teoz true
+
+skinparam backgroundColor #whitesmoke
+
+title Register Device Token
+
+box TransportSDK #lightcyan
+participant PushService
+participant WebMessagingAPI
+end box
+box Backend
+participant Thandora
+end box
+
+App -> PushService : synchronize(deviceToken, notificationProvider)
+PushService -> WebMessagingAPI : registerDeviceToken(pushConfig)
+WebMessagingAPI -> Thandora : POST /api/v2/push-notification/devices
+		rnote right of WebMessagingAPI
+            {
+            "tokenId": "0061a6ba-1r67-4ufe-8n2a-s6cc18907d81",
+            "deploymentId": "7322597b-802a-4159-ru03-n9a5873b5438",
```

Comment:
@AfanasievAnton 
How do you know what the deploymentId is? Don’t you also need to receive this as part of the synchronize(deviceToken, notificationProvider) method?

Reply:
In order to create PushService you need to create MessengerTransportSDK first, which requires Configuration object, which has deploymentId as a parameter.